source/dagger/gameplay/brawler/brawler_main.cpp

```diff
@@ -25,10 +25,14 @@
 #include "gameplay/brawler/systems/physics.h"
 #include "gameplay/brawler/systems/shooting_system.h"
 #include "gameplay/brawler/systems/weaponpickupsystem.h"
+#include "gameplay/brawler/systems/hud_system.h""
```

Comment:
There is an excess quote here. Why does it even compile?

Reply:
beats me